Microservices.PlatformService/appsettings.Production.json

```diff
@@ -0,0 +1,20 @@
+{
+  "ConnectionStrings": {
+    "MSSqlDatabaseProduction": "Server=mssql-clusterip-service,1433;Database=PlatformService;Initial Catalog=platformdb;User ID=sa;Password=pass55word!"
```

Comment:
Sensitive data in connection string.

Reply:
Yes, but it's production config.
Where I should store this data?